lib/fs-watch/src/either_stream.rs

```diff
@@ -5,6 +5,7 @@ use futures::{Poll, Stream};
 /// Combines two different `Stream`s yielding the same item and error
 /// types into a single type.
 #[derive(Clone, Debug)]
+#[allow(dead_code)]
```

Comment:
It's not dead code on Linux. Perhaps better would be to put the same cfg on the `mod either_stream` in `lib.rs`.

Reply:
Alternatively, we could add `#[cfg(target_os="linux")]` to the module definition.

Reply:
Sorry, I meant `#[cfg(target_os="linux")] mod either_stream`, just was on mobile and didn't want to type the symbols.